### ISSUE TITLE
More accessible report viewer

### ISSUE THREAD

**Issue by Kr0nox** [jplag/JPlag#2519]
A long term goal should be to make the report viewer more accessible.

This includes following the [WCAG Guidelines](https://www.w3.org/WAI/standards-guidelines/wcag/).

Doing so should improve the usability for many users and make it compatible with screen readers
